packages/storefront-core/tests/e2e/seed-product-filters.php

```
@@ -0,0 +1,108 @@
+
+if ( ! class_exists( 'WooCommerce' ) ) {
+	WP_CLI::error( 'WooCommerce must be active before seeding filter fixtures.' );
+}
+
+function bhaivatech_alpha_filter_product( string $name ): WC_Product {
+	foreach ( wc_get_products( array( 'limit' => -1 ) ) as $product ) {
+		if ( $product instanceof WC_Product && $product->get_name() === $name ) {
+			return $product;
+		}
+	}
+	WP_CLI::error( 'Missing filter fixture product: ' . $name );
```

PHPStan (2.2.7)

[error] Function bhaivatech_alpha_filter_product() should return WC_Product but return statement is missing.


(return.missing)